architectures/decentralized/solana-tooling/tests/suites/memnet_coordinator_rewards.rs

```diff
@@ -39,15 +39,15 @@ pub async fn run() {
     // Create payer key and fund it
     let payer = Keypair::new();
     endpoint
-        .process_airdrop(&payer.pubkey(), 5_000_000_000)
+        .request_airdrop(&payer.pubkey(), 5_000_000_000)
         .await
         .unwrap();
 
     // Run constants
     let main_authority = Keypair::new();
     let join_authority = Keypair::new();
     let mut clients = vec![];
-    for _ in 0..40 {
+    for _ in 0..240 {
```

Comment:
this was previously impossible to test with so many clients because the tests were too slow to run for 200+ clients, but it is now possible thanks to this PR we can test deeper the limits of the program.